src/main/java/org/apache/ibatis/migration/options/Options.java

```diff
@@ -15,6 +15,85 @@
  */
 package org.apache.ibatis.migration.options;
 
+import java.io.File;
+
 public enum Options {
-  PATH, ENVPATH, SCRIPTPATH, DRIVERPATH, HOOKPATH, ENV, FORCE, TRACE, HELP, TEMPLATE, IDPATTERN
+
```

Comment:
I use to have this like this if you look at the history but I split it out because I didn't want the Options enum to know about SelectedOptions etc. This is an implementation detail and splitting it out was to allow changing the implementation to something else such as using a cmd line parsing library instead.

Reply:
I see - 

Options don't need to know about "Selected" - put selected `Option`s in a `Set<Option> selected` and have each option apply itself when invoked. We can iterate on that a bit if you want